electra/the_fun_part/fun.c

```diff
@@ -763,6 +763,9 @@ do { \
             waitpid(pd, NULL, 0);
             symlink("/usr/lib/SBInject","/"BOOTSTRAP_PREFIX"/Library/SBInject");
         }
+        
+        mkdir("/Library/MobileSubstrate/DynamicLibraries", 0755);
```

Comment:
mkdir is not recursive. This will fail if /Library/MobileSubstrate doesn't exist

Reply:
fixing